src/components/forms/FormButton.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import PropTypes from "prop-types";
+import AppButton from "../AppButton";
+
+const FormButton = ({ title, onPress }) => {
+  return <AppButton title={title} onPress={onPress} />;
+};
+
+FormButton.propTypes = {
+  title: PropTypes.string.isRequired,
+  onPress: PropTypes.func.isRequired,
+};
+
+export default FormButton;
```

Comment:
Czy ten plik nie jest w tym momencie niepotrzebny ? W AppButton trzymane są style więc zmiana rodzica też tutaj sie spropaguje?

Reply:
to miało być na wypadek  gdyby przycisku submit w formularzu miał być bardziej rozbudowany ale w tym momencie faktycznie jest niepotrzebny więc wrzucę poprawkę w następnym commicie, najwyżej się go doda ponownie